frontend/src/views/WheresMyGene/common/types.ts

```diff
@@ -13,15 +14,25 @@ export interface CellType {
   name: string;
 }
 
+/**
+ * This object holds a gene and all the cell types that has this
```

Comment:
I'm not sure I understand what `all the cell types that has this genetic expression` means. Is this a list of cell types that have non zero expression levels for the gene?

Reply:
Thanks Maidons! That's right!

The object shape is:

```ts
export interface RawGeneExpression {
  gene_name: string;
  cell_types: GeneExpression[];
}
```

So each gene will have one `RawGeneExpression` object

E.g.,

geneA:

```ts
{
  gene_name: "geneA",
  cell_types: [ { id: "cellTypeIdA, pc: 0.01, me: 0.01 }, { id: "cellTypeIdB", pc: 0.02, me: 0.02 }]
}
```

geneB:

```ts
{
  gene_name: "geneB",
  cell_types: [ { id: "cellTypeIdA, pc: 0.03, me: 0.03 }, { id: "cellTypeIdB", pc: 0.04, me: 0.04 }]
}
```